assets/source/setup-guide/app/views/LandingPageApp.test.js

```diff
@@ -0,0 +1,90 @@
+jest.mock( '@woocommerce/tracks' );
+
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+import { fireEvent, render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import LandingPageApp from './LandingPageApp';
+import '../../../tests/custom-matchers';
```

Comment:
💅 

Totally out scope. But the more I see the code in pfw and gla, the more I'm wondering about  using[ jsconfig paths ](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360002490579-Custom-import-paths-jsconfig-json)

We can have something like `~tests/custom-matchers`, `~components/my-component`, `~data/reducer`... etc Instead of `../../../my-stuff`

Reply:
Oh, I thought we have it here, but it's only set for jest config https://github.com/woocommerce/pinterest-for-woocommerce/blob/add/232-setup-event/jest.config.js#L24:L24, but not actually aliased in webpack config.

I also vote to make the change we did in GLA (https://github.com/woocommerce/google-listings-and-ads/pull/192) here

Reply:
But, I'd rather make this change after we merge the tracking feature branch, to avoid merge conflicts.

Reply:
Created an issue at https://github.com/woocommerce/pinterest-for-woocommerce/issues/310